config/bundles.php

```diff
@@ -4,4 +4,6 @@
 
 return [
     Symfony\Bundle\FrameworkBundle\FrameworkBundle::class => ['all' => true],
+    Symfony\Bundle\TwigBundle\TwigBundle::class => ['all' => true],
+    Symfony\Bundle\WebProfilerBundle\WebProfilerBundle::class => ['dev' => true, 'test' => true],
```

Comment:
J'essayais de lancer le dump server comme indiqué ici : https://symfony.com/doc/current/components/var_dumper.html#the-dump-server

J'obtenais une erreur:

```console
$ make console CMD="server:dump"
```
```
  There are no commands defined in the "server" namespace.                                                                       
                                                                                                                                 
  You may be looking for a command provided by the "Debug Bundle" which is currently not installed. Try running "composer requi  
  re symfony/debug-bundle --dev".  
```

Il semble qu'il faille aussi installer debug-bundle:

https://symfony.com/doc/current/components/var_dumper.html#installation

> If using it inside a Symfony application, make sure that the DebugBundle has been installed (or run composer require --dev symfony/debug-bundle to install it).

J'ai donc lancé ceci :

```bash
make composer CMD="require --dev symfony/debug-bundle"
```

Et c'est bon, le dump server se lance

Reply:
J'ai rajouté le bundle. Ceci dit, je suis partisan d'installer uniquement ce dont on a besoin pour avoir le projet le plus léger possible.
Ici, on est sur une dépendance dev, du coup ça ne me dérange pas de la rajouter, l'impact sera minime. Et si vraiment on se rend compte qu'on ne l'utilise pas, rien ne nous empêche de retirer cette dépendance.

Reply:
Evidemment. Mon questionnement était sur si ce DebugBundle _devait_ être ajouté, ou pas. Si au quotidien tu ne l'utilisais pas dans tes projets Symfony précédents c'est peut-être qu'on en effet on allait pas en avoir besoin. J'ai juste trifouillé les recoins de ce que cette PR installe dans l'optique de prendre en main l'écosystème PHP/Symfony :yum: